app/views/passengers/_form.html.haml

```diff
@@ -21,12 +21,13 @@
   .field
     = form.label :wheelchair
     = form.check_box :wheelchair
-  .field
-    = form.label :active
-    = form.check_box :active
+  - if @current_user.admin?
+    .field
+      = form.label :active
+      = form.check_box :active
   .field
     = form.label :permanent_passenger
-    = form.check_box :permanent
+    = form.check_box :permanent, disabled: !@current_user.admin?
```

Comment:
If you're going to do this a lot, I'd recommend writing a predicate method for this.